clash-lib/src/Clash/Core/Evaluator.hs

```diff
@@ -356,11 +357,18 @@ force :: Heap -> Stack -> Id -> Maybe State
 force (Heap gh g@(GPureHeap gbl) h ids is) k x' = case lookupVarEnv x' h of
     Nothing -> case lookupVarEnv x' gbl of
       Just e | isGlobalId x'
-        -> Just (Heap gh (GPureHeap (delVarEnv gbl x')) h ids is,GUpdate x':k,e)
+        -> let e' = tickExpr e
+            in do traceM $ "force: " <> (Text.unpack . nameOcc $ varName x')
```

Comment:
^

Reply:
All of these traces will be removed for the final PR I assume?

Reply:
Ah right, I thought this was the final PR.